assets/queries/cloudFormation/LB_sensitive_port_is_exposed_to_entire_network/test/negative.yaml

```diff
@@ -0,0 +1,128 @@
+AWSTemplateFormatVersion: 2010-09-09
+Resources:
+  MyLoadBalancer:
+      Type: AWS::ElasticLoadBalancing::LoadBalancer
+      Properties:
+        AvailabilityZones:
+        - "us-east-2a"
+        CrossZone: true
+        Scheme: internet-facing
+        Listeners:
+        - InstancePort: '80'
+          InstanceProtocol: HTTP
+          LoadBalancerPort: '443'
+          Protocol: HTTPS
+          PolicyNames:
+          - My-SSLNegotiation-Policy
+          SSLCertificateId: arn:aws:iam::123456789012:server-certificate/my-server-certificate
+        HealthCheck:
+          Target: HTTP:80/
+          HealthyThreshold: '2'
+          UnhealthyThreshold: '3'
+          Interval: '10'
+          Timeout: '5'
+        SecurityGroups:
+          [ !Ref LBSecGroup ]
+        Policies:
+        - PolicyName: My-SSLNegotiation-Policy
+          PolicyType: SSLNegotiationPolicyType
+          Attributes:
+          - Name: Reference-Security-Policy
+            Value: ELBSecurityPolicy-TLS-1-2-2017-01
+  LBSecGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+        GroupDescription: Allow http and ssh
+        VpcId: my-vpc
+        SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: 22
+          ToPort: 22
+          CidrIp: 127.0.0.1/32
+        - IpProtocol: tcp
+          FromPort: 22
+          ToPort: 22
+          CidrIp: 127.0.0.1/33
+        SecurityGroupEgress:
+        - IpProtocol: tcp
+          FromPort: 22
+          ToPort: 22
+          CidrIp: 0.0.0.0/0
+
+---
+AWSTemplateFormatVersion: 2010-09-09
+Parameters:
+  Instance01:
+    Description: "instance 01"
+    Type: String
+  Instance02:
+    Description: "instance 02"
+    Type: String
+  MySubnet:
+    Description: "My subnet"
+    Type: List
+Resources:
+  NetworkLoadBalancer:
+    Type: AWS::ElasticLoadBalancingV2::LoadBalancer
+    Properties:
+      Name: t10-networkloadbalancer
+      Scheme: internet-facing
+      Subnets: !Ref MySubnet
+      SecurityGroups:
+        [ !Ref LBSecGroupV2 ]
+      Type: network
```

Comment:
Hi @AdarWeidman I'll probably have to change this since the linter gave me some errors: `E2503 Security groups are not supported for load balancers with type "network"`

Found [this documentation](https://aws.amazon.com/premiumsupport/knowledge-center/security-group-load-balancer/). I'll have to work around this with the samples trying to maintain the same query.
> If you're using a Network Load Balancer, update the security groups for your target instances, because Network Load Balancers do not have associated security groups.

cc: @ruigomescx